### PR TITLE
Two more nodes and fixes in the deps section

### PR DESCRIPTION
**Pull request by 2m**

### CODE REVIEW

File: infra/README.md [hunk 1]
```diff
@@ -6,24 +6,37 @@ Dependencies
...
+
+```
+mv berks-cookbooks cookbooks
+```
+
+Tryout the setup by running a command on a group of nodes:
+
+```
+bundle exec fix --concurrency 2 nodes_with_role:benchmark-ddata ssh:"hostname"
+```
```
Comment (ktoso) [inline infra/README.md]: good :)

**Comment by ktoso** [comment]
Good:)

File: infra/site-cookbooks/ab/metadata.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "ab",
-  "description": "Installs ab",
+  "description": "Installs/Configures ab",
   "long_description": "Installs/Configures ab",
-  "maintainer": "Konrad Malawski",
+  "maintainer": "YOUR_NAME",
```
Comment (ktoso) [inline infra/site-cookbooks/ab/metadata.json]: btw lol: https://twitter.com/ktosopl/status/875337843060621313

Comment (2m) [inline infra/site-cookbooks/ab/metadata.json]: Lol. :) Should add this file to .gitignore?

**Pull request merged**